src/components/NavBar/NavMenu.jsx

```diff
@@ -30,6 +32,7 @@ const NavMenu = ({
   profileImg
 }) => {
   const theme = useTheme();
+  const { setSelectedUser } = useContext(SelectedUserContext);
```

Comment:
With how these things are routing now, the `selectedUserContext` isn't actually needed anymore. The 'selected user' information is contained in the route. You select a user by navigating to their profile page. I think we can remove this, and probably the entire context.

Reply:
Have removed SelectedUserContext.